backend/census_historical_migration/workbooklib/notes_to_sefa.py

```diff
@@ -68,7 +68,10 @@ def xform_is_minimis_rate_used(rate_content):
         r"not\s+us(e|ed)",
         r"not\s+elec(t|ted)",
         r"not\s+applicable",
+        r"not\s+utilized",
         r"has\s+elected\s+not",
+        r"no\s+election",
+        r"not\s+(made|make)\s+(this|an)\s+election",
         r"did\s+not\s+charge\s+indirect\s+costs",
         r"did\s+not\s+make\s+an\s+election",
```

Comment:
`r"did\s+not\s+make\s+an\s+election"` can be removed now. `r"not\s+(made|make)\s+(this|an)\s+election"` is already covering it

Reply:
Done